bool enable
int32 ctrmode
int32 turnmode    # 0-阿克曼  1-差速模式  2-自转模式  3-横移模式  4-斜行    
float32[4] speed